applications/main/nfc/scenes/nfc_scene_saved_menu.c

```diff
@@ -75,21 +75,15 @@ void nfc_scene_saved_menu_on_enter(void* context) {
        !mf_ul_is_full_capture(&nfc->dev->dev_data.mf_ul_data)) {
         submenu_add_item(
             submenu,
-            "Unlock With Reader",
-            SubmenuIndexMfUlUnlockByReader,
-            nfc_scene_saved_menu_submenu_callback,
-            nfc);
-        submenu_add_item(
-            submenu,
-            "Unlock With Password",
+            "Unlock",
```

Comment:
Something went wrong here. Probably merge artifacts, please fix.

Reply:
I'm not a UI designer and I shouldn't have made such drastic changes. I updated it so that the only changes are in spelling, to match capitalization across the rest of the OS.